core/block_drag_surface.js

```diff
@@ -128,6 +130,16 @@ Blockly.BlockDragSurfaceSvg.prototype.createDom = function() {
       'filter', 'url(#' + this.dragShadowFilterId_ + ')');
 };
 
+/**
+ * Set the distance to offset the dragged SVG (in pixels).
+ * @param {number} left - Left offset.
+ * @param {number} top - Y offset.
+ */
+Blockly.BlockDragSurfaceSvg.prototype.setOffset = function(left, top) {
+  this.SVG_.style.marginLeft = left + 'px';
+  this.SVG_.style.marginTop = top + 'px';
```

Comment:
It may be [more performant](https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/) to use `transform` if that's possible here.